gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/impl/upgrade/IdentityProviderUpgrader.java

```diff
@@ -84,16 +85,18 @@ public boolean upgrade() {
                     if (id == null) {
                         id = type;
                     }
+
+                    String formattedId = IdGenerator.generate(id);
```

Comment:
It is a weird name for a function that "normalize" a string but 🤷

Reply:
💯 Agree! And it's also written in its Javadoc: `this method only apply a basic transformation and do not guaranty the uniqueness of id.`